docs/public/melange-cheatsheet.html

```diff
@@ -0,0 +1,676 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Melange Bindings Cheatsheet</title>
+  <style>
+    :root {
+      --c-bg: #ffffff;
+      --c-text: #213547;
+      --c-border: #ddd;
+      --c-code-bg: #f6f8fa;
+      --c-comment: #6a737d;
+    }
+
+    html.dark {
+      --c-bg: #1e1e20;
+      --c-text: #ddd;
+      --c-border: #383838;
+      --c-code-bg: #161618;
+      --c-comment: #9e9e9e;
+    }
+
+    body {
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+      margin: 0;
+      padding: 20px;
+      line-height: 1.5;
+      color: var(--c-text);
+      background-color: var(--c-bg);
+      transition: background-color 0.3s, color 0.3s;
+    }
+
+    .header {
+      text-align: center;
+      margin-bottom: 30px;
+      padding-bottom: 15px;
+    }
+
+    .header h1 {
+      margin: 0;
+      font-size: 32px;
+    }
+
+    .container {
+      display: grid;
+      grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+      gap: 20px;
+      max-width: 1600px;
+      margin: 0 auto;
+    }
+
+    .section {
+      break-inside: avoid;
+      margin-bottom: 20px;
+    }
+
+    h2 {
+      font-size: 24px;
+      margin-top: 0;
+      margin-bottom: 15px;
+      border-bottom: 1px solid var(--c-border);
+      padding-bottom: 5px;
+    }
+
+    .snippet {
+      margin-bottom: 15px;
+      font-size: 14px;
+    }
+
+    .comment {
+      color: var(--c-comment);
+      margin-bottom: 5px;
+    }
+
+    code {
+      font-family: SFMono-Regular, Consolas, "Liberation Mono", Menlo, monospace;
+      font-size: 13px;
+      background-color: var(--c-code-bg);
+      border-radius: 3px;
+      padding: 2px 4px;
+    }
+
+    pre {
+      background-color: var(--c-code-bg);
+      border-radius: 6px;
+      padding: 12px;
+      overflow-x: auto;
+      margin: 0;
+      transition: background-color 0.3s;
+    }
+
+    /* Syntax toggle styles */
+    .ocaml,
+    .reasonml {
+      display: none;
+    }
+
+    .syntax__ocaml .ocaml {
+      display: block;
+    }
+
+    .syntax__reasonml .reasonml {
+      display: block;
+    }
+  </style>
+</head>
+
+<body>
```

Comment:
is there any chance to use the regular header here? or treat this page as a .md file, etc?

Re-creating everything might cause future updates to be more painful